src/components/ModalAddManyTokens.js

```diff
@@ -13,10 +13,11 @@ import hathorLib from '@hathor/wallet-lib';
 import { connect } from 'react-redux';
 import tokens from '../utils/tokens';
 import wallet from "../utils/wallet";
+import { getGlobalWallet } from "../services/wallet.singleton";
 
 
 const mapStateToProps = (state) => {
-  return { storage: state.wallet.storage };
+  return { storage: getGlobalWallet().storage };
```

Comment:
I understand that this is simpler, but why not just remove the react-redux `connect` and use the `getGlobalWallet` directly where it is used?

The idea of `mapStateToProps` is to map something on state to the component props, but the idea of this PR is to remove the wallet from the state.

Reply:
Makes sense. Refactored on https://github.com/HathorNetwork/hathor-wallet/pull/580/commits/b4224ead29f465e482ae67410da896ca794a5e68.